pandas/tests/indexing/test_loc.py

```diff
@@ -3297,3 +3297,43 @@ def test_loc_reindexing_of_empty_index(self):
         df.loc[Series([False] * 4, index=df.index, name=0), 0] = df[0]
         expected = DataFrame(index=[1, 1, 2, 2], data=["1", "1", "2", "2"])
         tm.assert_frame_equal(df, expected)
+
+    def test_loc_set_series_to_multiple_columns(self):
```

Comment:
Can you parametrize this test  using `@pytest.mark.parametrize` - ideally there would be one case per test run. 

Tests that run multiple cases are difficult to debug because they may fail in the middle and you hit break points multiple times for other cases.

Reply:
Done